cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSICKANSink.java

```diff
@@ -66,7 +67,7 @@
     private String ckanViewer;
     private CKANBackend persistenceBackend;
 
-    /**
+    /*entityId
```

Comment:
```suggestion
    /**
```

Seems to be a typo

Reply:
Fixed in [0ab256e](https://github.com/telefonicaid/fiware-cygnus/pull/1850/commits/0ab256e454a652c3ec81c53c46be1cc41a9a0830)